cmd/goal/account.go

```diff
@@ -129,11 +127,16 @@ func init() {
 	changeOnlineCmd.Flags().BoolVarP(&online, "online", "o", true, "Set this account to online or offline")
 	changeOnlineCmd.MarkFlagRequired("online")
 	changeOnlineCmd.Flags().Uint64VarP(&transactionFee, "fee", "f", 0, "The Fee to set on the status change transaction (defaults to suggested fee)")
-	changeOnlineCmd.Flags().Uint64VarP(&statusChangeFirstRound, "firstRound", "", 0, "FirstValid for the status change transaction (0 for current)")
-	changeOnlineCmd.Flags().Uint64VarP(&statusChangeValidRounds, "validRounds", "v", 0, "The validity period for the status change transaction")
+	changeOnlineCmd.Flags().Uint64VarP(&firstValid, "firstRound", "", 0, "FirstValid for the status change transaction (0 for current, deprecated)")
+	changeOnlineCmd.Flags().Uint64VarP(&firstValid, "firstvalid", "", 0, "The first round where the status change transaction may be committed to the ledger (0 for current)")
+	changeOnlineCmd.Flags().Uint64VarP(&numValidRounds, "validRounds", "", 0, "The validity period for the status change transaction (deprecated)")
+	changeOnlineCmd.Flags().Uint64VarP(&numValidRounds, "validrounds", "v", 0, "The validity period for the status change transaction")
```

Comment:
I guess that this is for backwards compatibility? I would really prefer if we push back on whatever push back you received and make sure the doc's are updated rather than polluting the interface.

Reply:
Seems we want to keep them at least for current release.